pwa/components/dashboard/DashboardLayout.tsx

```diff
@@ -191,7 +191,7 @@ const DashboardLayout = ({children}: DashboardLayoutProps) => {
                     )}
                   </>
                 )}
-                {(repairer ||
+                {((repairer && repairer.enabled) ||
                   (user &&
                     user.repairerEmployee &&
                     user.repairerEmployee.repairer)) && (
```

Comment:
I don't know if you have to check if `user.repairerEmployee.repairer` is enabled too?